### PR TITLE
Added fixes to append public or private keywords

### PR DESCRIPTION
**Pull request by limxingzhi**
- appended private keyword to fix own props
- appended private keyword to fix own methods
- appended public keyword to fix decorated props

### CODE REVIEW

**Comment by StevenMeyer** [comment]
Also is adding `private ` the fix? The ES private `#` is private, too.

File: src/rules/props-must-be-public.ts [hunk 1]
```diff
@@ -25,7 +26,10 @@ const rule: Rule.RuleModule = {
           if (isPrivate(originalNode)) {
             context.report({
               node: node,
-              message: `Class properties decorated with @Prop() cannot be private nor protected`
+              message: `Class properties decorated with @Prop() cannot be private nor protected`,
```
[the remark this answers is not in the document]
Comment (limxingzhi) [reply src/rules/props-must-be-public.ts:28]: @rwaskiewicz SFLR, both concerns are addressed

**Comment by raphaelpor** [comment]
Hey @rwaskiewicz,
Do you still think this change is valid? Just asking since you have started reviewing it.
If so, I can proceed with the testing/merging. :)

**Comment by rwaskiewicz** [comment]
@raphaelpor Yes, I think this still looks good. Comparing the current changes to what I have on @limxingzhi's branch from when I initially reviewed, it looks like all the feedback has been addressed. I haven't done any testing, but it looks good to me from the standpoint of my initial review 👍

**Comment by raphaelpor** [comment]
@rwaskiewicz Great! I will take a look and merge it for the next release. 

Thanks! ✌️

**Review by raphaelpor — APPROVED** [review]

**Comment by raphaelpor** [comment]
@limxingzhi Hello and thanks for your contribution.

Sorry for taking so long to review your code. But looks like the tests are failing, now.
Get we get some help from you or someone else to fix them?

**Comment by limxingzhi** [comment]
It has been over a year since I worked on this tech stack, I would be more than happy if someone took over

**Pull request merged**